src/client/firefox/commands.js

```diff
@@ -121,7 +121,7 @@ function setBreakpoint(
     })
     .then(([{ actualLocation }, bpClient]) => {
       actualLocation = createBreakpointLocation(location, actualLocation);
-      const id = makeLocationId(actualLocation);
+      const id = makePendingLocationId(actualLocation);
```

Comment:
where is this coming from :)

looks like this PR desperately needs a rebase :arrow_heading_up: :arrow_heading_down: